src/json/parse.js

```diff
@@ -21,7 +24,6 @@ let text;
 
 const error = function(m) {
   // Call error when something is wrong.
-
   throw {
     name: 'SyntaxError',
     message: m,
```

Comment:
Unrelated to this PR. What do you think of this rule btw: https://eslint.org/docs/rules/no-throw-literal

Reply:
I agree I follow that too. But, this part came from Douglas Crockford's JSON code. I was planning to add eslint anyways. Will update these then. Thanks for pointing it out. I didn't notice it 👍